E5_documentation.md

```diff
@@ -0,0 +1,13 @@
+# Developer's guide: updating the documentation
+
+Everybody is welcome to update/extend the documentation.
+
+For doing so:
+
+- clone the documentation [repository](https://github.com/intelligent-soft-robots/pam_documentation)
+- update the documentation
+- If you add a new file: note that there is no need for updating an index or a the list of contents. This is done automatically.
+- If you add a new file: in the content list, the orders of the page is alphabetical. Name your new markdown file based on the order into which it should appear in the content list.
```

Comment:
This is not the case anymore, there is now an `index.rst` that needs to be maintained manually.

Reply:
I would also suggest to use RST for new files, as I think it is better suited for larger documentations (e.g. offers labels that can be referenced and things like "info"/"warning" boxes).